lib/oli/delivery/paywall.ex

```diff
@@ -511,6 +528,9 @@ defmodule Oli.Delivery.Paywall do
       iex> create_or_update_discount(discount, %{name: bad_value})
       {:error, %Ecto.Changeset{}}
   """
+  def create_or_update_discount(%{institution_id: nil} = attrs),
+    do: {:error, Ecto.Changeset.add_error(Discount.changeset(%Discount{}, attrs), :institution, "can't be blank")}
```

Comment:
It is, but since it's a create or update (and look for discounts depending on the institution), I did it this way to be more defensive. But we can actually change the function to not be create or update if that's not clear at all

Reply:
After some discussion offline...

If we try to compare with nil, Ecto will raise:
![image](https://user-images.githubusercontent.com/42446726/171952481-82c9c659-9e90-459a-a550-9a7459b5fde9.png)

That's why I opted to match this way